mmv1/products/firebasedatabase/terraform.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Instance: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: ["projects/{{project_id}}/locations/{{location_id}}/instances/{{database_id}}"]
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 10
+      update_minutes: 10
+    skip_delete: true
+    skip_sweeper: true
```

Comment:
I was primarily following the example [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/864ca08a03d5facd1811617f483f320407dce12c/mmv1/products/firebase/terraform.yaml#L22).

Also, Databases cannot be deleted unless they are first disabled, which is a separate RPC, which I wasn't sure how to handle. I didn't realize that we reused projects from test to test, do you have any advice on how to best handle this.

Reply:
Hmm, I think we're probably fine on the quota issue actually - the test projects do get cleaned up. As long as the Firebase database gets automatically cleaned up when the project is deleted we should be fine.

Regarding how to support disabling/deletion - it looks like it might make sense to support the `state` field and update it with the disable / re-enable API endpoints? This would be sort of similar to how [updating project billing](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/third_party/terraform/resources/resource_google_project.go#L514) works... but maybe there's a different way to implement it that doesn't require changing the whole resource. I'll check with the team.